python3/electro_grammar/electro_grammar.py

```diff
@@ -0,0 +1,48 @@
+from antlr4 import *
+from antlr4.error.DiagnosticErrorListener import DiagnosticErrorListener
+from antlr4.atn.PredictionMode import PredictionMode
+from decimal import Decimal
+from electro_grammar.ElectroGrammarLexer import ElectroGrammarLexer
+from electro_grammar.ElectroGrammarListener import ElectroGrammarListener
+from electro_grammar.ElectroGrammarParser import ElectroGrammarParser
+
+
+class ElectroGrammarToObjectListener(ElectroGrammarListener):
+    def __init__(self):
+        self.obj = {}
+
+    def exitVoltage(self, ctx):
+        number = Decimal(str(ctx.NUMBER()))
+        self.obj['voltage'] = number;
+
+    def enterResistance(self, ctx):
+        rprefix_lookup = {'M': 10e6, 'k': 10e3, 'K': 10e3, 'None': 1, 'm': 10-3};
+        number = Decimal(str(ctx.NUMBER()))
+        #print(str(ctx.RPREFIX()))
+        #rprefix = rprefix_lookup[str(ctx.RPREFIX())]
+        self.obj['resistance'] = number #* rprefix
+
+    def enterCapacitance(self, ctx):
+        #cprefix_lookup = {'u': 10e-6, 'n': 10e-9, 'p': 10e-12}
```

Comment:
By the way, your calculations are a little off. 1uF is 1e-6 not 10e-6, etc. I fixed this in the JS version.